bin/alluxio-start.sh

```diff
@@ -172,19 +169,6 @@ start_master() {
     fi
   fi
 
-  if [[ ${ALLUXIO_MASTER_SECONDARY} == "true" ]]; then
-    if [[ `${LAUNCHER} ${BIN}/alluxio getConf ${ALLUXIO_MASTER_JAVA_OPTS} alluxio.master.journal.type` == "EMBEDDED" ]]; then
-      echo "Secondary master is not supported for journal type: EMBEDDED"
-      exit 1
-    fi
-    echo "Starting secondary master @ $(hostname -f). Logging to ${ALLUXIO_LOGS_DIR}"
-    (nohup ${BIN}/launch-process secondary_master > ${ALLUXIO_LOGS_DIR}/secondary_master.out 2>&1) &
-  else
-    echo "Starting master @ $(hostname -f). Logging to ${ALLUXIO_LOGS_DIR}"
-    (JOURNAL_BACKUP="${journal_backup}" nohup ${BIN}/launch-process master > ${ALLUXIO_LOGS_DIR}/master.out 2>&1) &
```

Comment:
keep the else branch... need to start normal master

Reply:
oops